src/main/resources/io/jenkins/plugins/designlibrary/Spacing/index.jelly

```diff
@@ -1,34 +1,18 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples">
-  <s:sample>
-    <h2 class="jdl-heading">${%title}</h2>
+  <s:layout>
+    <s:section hideBorder="true">
+      <s:preview>
+        <div class="jdl-padding-box jdl-padding-box--5">
+          <p class="jdl-paragraph">.jenkins-!-padding-5</p>
+        </div>
+      </s:preview>
 
-    <p class="jdl-paragraph">${%description.1}</p>
+      <p class="jdl-paragraph">${%description.1}</p>
 
-    <p class="jdl-paragraph">${%description.2}</p>
+      <p class="jdl-paragraph">${%description.2}</p>
 
-    <p class="jdl-paragraph">${%description.3}</p>
-
-    <h3>${%examples}</h3>
-
-    <div class="jdl-component-sample">
```

Comment:
I think it would be good to have more examples on this page, its too texty to me really, instead of saying `jenkins-!-margin-*` it would be good to have examples

Reply:
Added a little interactive sample to the page:

https://github.com/user-attachments/assets/633119ab-2367-4ab9-9c42-29c986fed313

Reply:
video doesn't seem to work for me

Reply:
looks nice though